nanoFramework.Json.Benchmark/nanoFramework.Json.Benchmark.nfproj

```diff
@@ -17,6 +17,7 @@
     <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
     <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
     <RestoreLockedMode Condition="'$(TF_BUILD)' == 'True' or '$(ContinuousIntegrationBuild)' == 'True'">true</RestoreLockedMode>
+    <CodeAnalysisRuleSet>..\.sonarlint\nanoframework_lib-nanoframework.jsoncsharp.ruleset</CodeAnalysisRuleSet>
```

Comment:
I don't see any .sonarlint folder in the repository:
![image](https://user-images.githubusercontent.com/8145578/198291072-b287de52-de44-4da1-a93c-bef593c88901.png)

Especially if you add the repository in the .gitignore :-)
![image](https://user-images.githubusercontent.com/8145578/198291173-4b6599e0-aa17-4261-bc41-50ad71803f9c.png)

So something not adding up somewhere!

Reply:
You are correct: these should have been checked in. It was an experiment... 🤦🏻‍♂️ I'll fix that.

I'll also add a short instruction on how to use this and add it to our docs collection.

Basically, you have to install [SonarLint VS extension](https://marketplace.visualstudio.com/items?itemName=SonarSource.SonarLintforVisualStudio2022), configure it once, and then you'll have the project specific ruleset available locally.